libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/BuildingExtrusionLayer.java

```diff
@@ -91,6 +94,33 @@ public void onStyleLoaded(@NonNull Style style) {
     return layerVisible;
   }
 
+  /**
+   * Retrieve the latest set opacity of the building extrusion layer.
+   *
+   * @return the opacity Float
+   */
+  public Float getOpacity() {
+    return opacity;
```

Comment:
This and all exposed fields below will return `0`, if the developer is using default values. These methods should either query the map for the actual value, or the fields should be initialized with default values.

Reply:
Switched to 

```java
private static final Float DEFAULT_FILL_EXTRUSION_OPACITY = 0.6f;
private static final Integer DEFAULT_FILL_EXTRUSION_COLOR = Color.parseColor("#F9F9F9");
private static final Float DEFAULT_FILL_EXTRUSION_MIN_ZOOM_LEVEL = 15.5f;
private Float opacity = DEFAULT_FILL_EXTRUSION_OPACITY;
private Integer color = DEFAULT_FILL_EXTRUSION_COLOR;
private Float minZoomLevel = DEFAULT_FILL_EXTRUSION_MIN_ZOOM_LEVEL;
```